packages/trader/src/Modules/SmartChartBeta/Components/LaunchModal/__tests__/launch-modal.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import LaunchModal from '../launch-modal';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+describe('Launch Modal', () => {
+    let modal_root_el: HTMLDivElement;
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    beforeEach(() => {
+        // To clear localStorage on each test run
+        localStorage.removeItem('launchModalShown');
+    });
+
+    it('should display launch modal for a logged in user', async () => {
+        const mocked_store = mockStore({
+            client: { is_logged_in: true },
+        });
+
+        render(
+            <StoreProvider store={mocked_store}>
+                <LaunchModal />
+            </StoreProvider>
+        );
+        expect(screen.getByTestId('launch-modal')).toBeInTheDocument();
+    });
+
+    it('should not display launch modal for a not logged in user', async () => {
+        const mocked_store = mockStore({
+            client: { is_logged_in: false },
+        });
+
+        render(
+            <StoreProvider store={mocked_store}>
+                <LaunchModal />
+            </StoreProvider>
+        );
+        expect(screen.queryByTestId('launch-modal')).not.toBeInTheDocument();
+    });
+
+    it('should set the localStorage key launchModalShown to true on clicking the continue button', async () => {
+        const mocked_store = mockStore({
+            client: { is_logged_in: true },
+        });
+        render(
+            <StoreProvider store={mocked_store}>
+                <LaunchModal />
+            </StoreProvider>
+        );
+        const continueBtn = screen.getByRole('button', { name: 'Continue' });
```

Comment:
```suggestion
        const continue_btn = screen.getByRole('button', { name: 'Continue' });
```
@ahmadtaimoor-deriv please use snake_case as per naming convention on the project :)